docker/etna-base/entrypoints/build.sh

```diff
@@ -70,3 +70,7 @@ fi
 if [ -n "$RUN_NPM_INSTALL" ]; then
   if [ -e ../etna ]; then npm link ../etna/packages/etna-js; fi
 fi
+
+if [ "$APP_NAME" = "timur" ]; then
```

Comment:
Hmm, so this actually already does exist in `timur/build/create-routes` is it not being run?

Reply:
Hm, I don't see a folder `timur/build` or a script at `timur/build/create-routes` in the repo, and if I grep for `create_routes` or try a `find . -name create-routes` I get no results other than the actual task itself ... sorry, am I just not seeing it?